contracts/oracles/FraxOracle.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+import { OracleInterface } from "../interfaces/OracleInterface.sol";
+import { ISFrax } from "../interfaces/ISFrax.sol";
+import { ISfraxETH } from "../interfaces/ISfraxETH.sol";
+import { ensureNonzeroAddress } from "@venusprotocol/solidity-utilities/contracts/validators.sol";
+import { EXP_SCALE } from "@venusprotocol/solidity-utilities/contracts/constants.sol";
+
+/**
+ * @title FraxOracle
+ * @author Venus
+ * @notice This oracle fetches the price of sFrax and sfrxETH assets
+ */
+contract FraxOracle is OracleInterface {
+    /// @notice Address of sfraxETH
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    ISfraxETH public immutable sfraxETH;
+
+    /// @notice Address of FRAX
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    address public immutable FRAX;
+
+    /// @notice Address of sFrax
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    ISFrax public immutable sFRAX;
+
+    /// @notice Address of Resilient Oracle
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    OracleInterface public immutable RESILIENT_ORACLE;
+
+    /// @notice Address of WETH (on Ethereum chain) or ETH (on BNB chain)
+    address public immutable ETH;
+
+    /// @notice Constructor for the implementation contract.
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address frax, address sFrax, address eth, address _sfraxETH, address resilientOracleAddress) {
```

Comment:
If we are using underscore in the param name then we should use it in all the params.